### DISCUSSION TITLE
Widgets

### DISCUSSION THREAD

**Discussion by xavierlvll** [fast4x/RiMusic#200] [Ideas]
Would it be possible to add widgets?

**Comment by fast4x** [comment]
That would be interesting, write down how they’re supposed to work, and we’ll figure it out.

**Comment by xavierlvll** [comment]
A couple of ideas and ways widgets could work: having the Quick Picks section as a widget to quickly play a song, and maybe having a widget for the player

An interesting one might be a 'play random song' or maybe a radio widget. Other than that, I can't think of any more right now.

**Comment by ALMIGHTY-VC** [comment]
https://youtu.be/YX5-8bgKLcg?si=iYa3NQxcvxK5tJ4B

**Reply by ALMIGHTY-VC** [reply]
This can serve as a idea too, i know their are crashes happening with many users and believe it would be better to release an alpha / beta version before stable releases
